Sources/DocCDocumentation/EmptyFile.swift

```diff
@@ -0,0 +1,10 @@
+//
+//  EmptyFile.swift
+//  
+//
+//  Created by Andrés Boedo on 3/28/22.
+//
+// This is an empty source file used to make RevenueCat a valid
+// documentation build target.
+//
+// see Apple's example at https://github.com/apple/swift-docc-plugin/
```

Comment:
I shamelessly ripped off the file structure that Apple's using in that repo ^^. 
But then again, they're using the plugin to publish the docs for the plugin, so we know it works. I also tested it out and it works correctly